### ISSUE TITLE
IBert Problems of hugging face pretrained

### ISSUE THREAD

**Issue by CSuperlei** [huggingface/transformers#14176]
### Pretrained the IBert
I want to test IBERT's, and I have done exactly what is said in https://huggingface.co/kssteven/ibert-roberta-base. For the quantization part, when I set quant_mode to true and run the evaluation again, I get a much low accuracy model.  What am I doing wrong?

**Comment by wenhuach** [comment]
I met the same issue on squad dataset.

**Comment by cjg91** [comment]
## Poor accuracy after finetuning

I follow the exact instruction in the [model card ](https://huggingface.co/kssteven/ibert-roberta-base)
`python examples/text-classification/run_glue.py \
         --model_name_or_path kssteven/ibert-roberta-base \
         --task_name MRPC \
         --do_eval \
         --do_train \
         --evaluation_strategy epoch \
         --max_seq_length 128 \
         --per_device_train_batch_size 32 \
         --save_steps 115 \
         --learning_rate 2e-5 \
         --num_train_epochs 10 \
         --output_dir $OUTPUT_DIR`
For finetuning on MRPC and then

`python examples/pytorch/text-classification/run_glue.py \
    --model_name_or_path ../output_dir/checkpoint-575/  \
    --task_name MRPC  \
    --do_eval        \
    --do_train        \
    --evaluation_strategy epoch    \
    --max_seq_length 128    \
    --per_device_train_batch_size 8        \
    --save_steps 115      \
    --learning_rate 1e-6      \
    --num_train_epochs 10      \
    --output_dir $OUTPUT_DIR`

for quantization-aware finetuning. Note that I lowered batch size since after enabling quantization my GPU memory could not handle the batch size of 32...

Result is horrible accuracy:
 | Training Loss | Epoch | Step | Validation Loss | Accuracy | F1  | Combined Score |
 |:-------------:|:-----:|:----:|:---------------:|:--------:|:---:|:--------------:|
 | No log        | 1.0   | 230  | 0.6933          | 0.3162   | 0.0 | 0.1581         |
